integration-tests/features/steps/stack_analyses_v2.py

```diff
@@ -270,6 +458,23 @@ def send_request(context, ecosystem=None, manifest=None, token='without', valid=
     post_request(context, ecosystem, manifest, with_user_key, with_valid_user_key)
 
 
+@when('I start {ecosystem} package request with manifest {manifest} '
```

Comment:
As mentioned before, let parameterized existing function to handle show_transitive flag.